archive/schema.go

```diff
@@ -28,9 +28,15 @@ type MetaData struct {
 	Spans             []SpanInfo     `json:"spans"`
 }
 
+type LogID string
+
+func (l LogID) Path(ark *Archive) string {
+	return filepath.Join(ark.Root, filepath.ToSlash(string(l)))
```

Comment:
Passing the result of filepath.ToSlash to filepath.Join doesn't seem right. Did you mean path.Join?